packages/tizen_log/lib/tizen_log.dart

```diff
@@ -0,0 +1,166 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:flutter/foundation.dart';
+
+typedef _DlogNative = Void Function(
+    Int32, Pointer<Utf8>, Pointer<Utf8>, Pointer<Utf8>);
+typedef _Dlog = void Function(int, Pointer<Utf8>, Pointer<Utf8>, Pointer<Utf8>);
+
+/// Provides the ability to use Tizen dlog logging service.
+/// More info: https://developer.tizen.org/dev-guide/training/native-app/en/wearable/lesson_13/index.html
+class Log {
+  Log._();
+
+  static final DynamicLibrary _library = DynamicLibrary.open('liblog.so');
```

Comment:
```
══╡ EXCEPTION CAUGHT BY GESTURE ╞═══════════════════════════════════════════════════════════════════
The following ArgumentError was thrown while handling a gesture:
Invalid argument(s): Failed to load dynamic library 'liblog.so': liblog.so: cannot open shared
object file: No such file or directory

When the exception was thrown, this was the stack:
#0      _open (dart:ffi-patch/ffi_dynamic_library_patch.dart:12:43)
#1      new DynamicLibrary.open (dart:ffi-patch/ffi_dynamic_library_patch.dart:23:12)
#2      Log._library (package:tizen_log/tizen_log.dart:19:57)
#3      Log._library (package:tizen_log/tizen_log.dart)
#4      Log._dlogPrint (package:tizen_log/tizen_log.dart:21:7)
#5      Log._dlogPrint (package:tizen_log/tizen_log.dart)
...
Handler: "onTap"
Recognizer:
  TapGestureRecognizer#dff4e
════════════════════════════════════════════════════════════════════════════════════════════════════
```

Did you mean
```suggestion
  static final DynamicLibrary _library = DynamicLibrary.open('libdlog.so.0');
```

Reply:
Thank you for noticing this, I wrongly assumed `liblog.so` is present on all devices.